### ISSUE TITLE
Unable to use EvolvingGraphs Package

### ISSUE THREAD

**Issue by ravitomar7** [EtymoIO/EvolvingGraphs.jl#11]
Hi,
I just installed Julia and then as instructed in manual 
 **Pkg.add("EvolvingGraphs")**

But when I use the **g = evolving_graph(Char, Int)**
I Get an Error **ERROR: UndefVarError: evolving_graph not defined**

Kindly help

**Comment by jpfairbanks** [comment]
did you use "using EvolvingGraphs"? Pkg.add installs the package and then `using Packagename` loads it into the current process.

**Comment by ravitomar7** [comment]
I have tried sir, Please find the snippet below:

julia> Pkg.add("EvolvingGraphs")
INFO: Package EvolvingGraphs is already installed
INFO: METADATA is out-of-date — you may not have the latest version of EvolvingGraphs
INFO: Use `Pkg.update()` to get the latest versions of your packages

julia> using EvolvingGraphs

julia> g = evolving_graph()
ERROR: UndefVarError: evolving_graph not defined

**Comment by jpfairbanks** [comment]
Which version of Julia? i just tried on 0.6 and got 

```julia
julia> using EvolvingGraphs
WARNING: could not import Base.slice into EvolvingGraphs

WARNING: deprecated syntax "abstract AbstractGraph{V,T,E}" at /root/.julia/v0.6/EvolvingGraphs/src/base.jl:9.
Use "abstract type AbstractGraph{V,T,E} end" instead.

WARNING: deprecated syntax "abstract AbstractEvolvingGraph{V,T,E}<:AbstractGraph{V,T,E}" at /root/.julia/v0.6/EvolvingGraphs/src/base.jl:10.
Use "abstract type AbstractEvolvingGraph{V,T,E}<:AbstractGraph{V,T,E} end" instead.

WARNING: deprecated syntax "abstract AbstractStaticGraph{V,E}<:AbstractGraph{V,E}" at /root/.julia/v0.6/EvolvingGraphs/src/base.jl:11.
Use "abstract type AbstractStaticGraph{V,E}<:AbstractGraph{V,E} end" instead.

WARNING: deprecated syntax "abstract AbstractPath" at /root/.julia/v0.6/EvolvingGraphs/src/base.jl:19.
Use "abstract type AbstractPath end" instead.

WARNING: deprecated syntax "typealias NodeType{V} Union{Node{V},AttributeNode{V},TimeNode{V}}" at /root/.julia/v0.6/EvolvingGraphs/src/base.jl:84.
Use "NodeType{V} = Union{Node{V},AttributeNode{V},TimeNode{V}}" instead.

WARNING: deprecated syntax "typealias NodeVector{V} Vector{Node{V}}" at /root/.julia/v0.6/EvolvingGraphs/src/base.jl:87.
Use "NodeVector{V} = Vector{Node{V}}" instead.

WARNING: deprecated syntax "typealias EdgeType{V} Union{Edge{V},TimeEdge{V},WeightedTimeEdge{V}}" at /root/.julia/v0.6/EvolvingGraphs/src/base.jl:137.
Use "EdgeType{V} = Union{Edge{V},TimeEdge{V},WeightedTimeEdge{V}}" instead.
ERROR: LoadError: LoadError: invalid subtyping in definition of AbstractStaticGraph
Stacktrace:
 [1] include_from_node1(::String) at ./loading.jl:569
 [2] include(::String) at ./sysimg.jl:14
 [3] include_from_node1(::String) at ./loading.jl:569
 [4] eval(::Module, ::Any) at ./boot.jl:235
 [5] _require(::Symbol) at ./loading.jl:483
 [6] require(::Symbol) at ./loading.jl:398
while loading /root/.julia/v0.6/EvolvingGraphs/src/base.jl, in expression starting on line 10
while loading /root/.julia/v0.6/EvolvingGraphs/src/EvolvingGraphs.jl, in expression starting on line 45
```

**Comment by ravitomar7** [comment]
I am also using version 0.6

Ravis-MacBook-Pro:~ ravitomar$ exec '/Applications/Julia-0.6.app/Contents/Resources/julia/bin/julia'
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: https://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.6.0 (2017-06-19 13:05 UTC)
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release
|__/                   |  x86_64-apple-darwin13.4.0

julia> using EvolvingGraphs
WARNING: could not import Base.slice into EvolvingGraphs

WARNING: deprecated syntax "abstract AbstractGraph{V,T,E}" at /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/base.jl:9.
Use "abstract type AbstractGraph{V,T,E} end" instead.

WARNING: deprecated syntax "abstract AbstractEvolvingGraph{V,T,E}<:AbstractGraph{V,T,E}" at /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/base.jl:10.
Use "abstract type AbstractEvolvingGraph{V,T,E}<:AbstractGraph{V,T,E} end" instead.

WARNING: deprecated syntax "abstract AbstractStaticGraph{V,E}<:AbstractGraph{V,E}" at /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/base.jl:11.
Use "abstract type AbstractStaticGraph{V,E}<:AbstractGraph{V,E} end" instead.

WARNING: deprecated syntax "abstract AbstractPath" at /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/base.jl:19.
Use "abstract type AbstractPath end" instead.

WARNING: deprecated syntax "typealias NodeType{V} Union{Node{V},AttributeNode{V},TimeNode{V}}" at /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/base.jl:84.
Use "NodeType{V} = Union{Node{V},AttributeNode{V},TimeNode{V}}" instead.

WARNING: deprecated syntax "typealias NodeVector{V} Vector{Node{V}}" at /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/base.jl:87.
Use "NodeVector{V} = Vector{Node{V}}" instead.

WARNING: deprecated syntax "typealias EdgeType{V} Union{Edge{V},TimeEdge{V},WeightedTimeEdge{V}}" at /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/base.jl:137.
Use "EdgeType{V} = Union{Edge{V},TimeEdge{V},WeightedTimeEdge{V}}" instead.
ERROR: LoadError: LoadError: invalid subtyping in definition of AbstractStaticGraph
Stacktrace:
 [1] include_from_node1(::String) at ./loading.jl:569
 [2] include(::String) at ./sysimg.jl:14
 [3] include_from_node1(::String) at ./loading.jl:569
 [4] eval(::Module, ::Any) at ./boot.jl:235
 [5] _require(::Symbol) at ./loading.jl:483
 [6] require(::Symbol) at ./loading.jl:398
while loading /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/base.jl, in expression starting on line 10
while loading /Users/ravitomar/.julia/v0.6/EvolvingGraphs/src/EvolvingGraphs.jl, in expression starting on line 45

**Comment by alexmorley** [comment]
Should be fixed in #12
